scripts/create_py_tags.py

```diff
@@ -8,296 +7,368 @@
 #
 # This script should be run in the top source directory.
 #
-# Parses all files given on command line for Python classes or functions and write
-# them into data/tags/std.py.tags (internal tagmanager format).
+# Parses all files in the directories given on command line for Python classes or functions and
+# write them into data/tags/std.py.tags (internal tagmanager format).
 # If called without command line arguments, a preset of common Python libs is used.
 #
 # WARNING
-# Be aware that running this script will actually *import* modules in the specified directory
+# Be aware that running this script will actually *import* all modules given on the command line
 # or in the standard library path of your Python installation. Dependent on what Python modules
 # you have installed, this might not be want you want and can have weird side effects.
```

Comment:
[…] what* you want […]

not that it changed in this PR though

Reply:
Amazing, someone actually read the docs :). Thank you for spotting, fixed.